functions.php

```diff
@@ -530,8 +530,18 @@ function ($canonical_url, $post) {
 
 if (class_exists('\\Sentry\\Options')) {
     add_filter('wp_sentry_options', function (\Sentry\Options $options) {
-        // Only sample 25% of the events
-        $options->setSampleRate(0.25);
+        // Only sample 50% of the events
+        $options->setSampleRate(0.50);
+
+        // Set server_name tag
+        $podname = gethostname();
+        $parts = explode('-', $podname);
+        $server_name = $parts[1];
```

Comment:
On local dev env, `gethostname()` function returns something like `7648d91278b7`(without  `-`) which leads a `PHP Warning: Undefined array key 1...` on this line, we can avoid it by -
```
$server_name = isset($parts[1]) ? $parts[1] : $podname;
```